### ISSUE TITLE
Apparently missing file

### ISSUE THREAD

**Issue by mxmilkb** [localnet/aurci#6]
carla-git is failing to build, with a missing file error, though that file is present in the Carla repo and carla-git build locally.

https://travis-ci.com/mxmilkb/aur-av-bin

**Comment by localnet** [comment]
After a quick look, I can see `carlastyle.json` at:

[source/theme/styles/](https://github.com/falkTX/Carla/blob/develop/source/theme/styles/)

but not at:

[source/theme/](https://github.com/falkTX/Carla/blob/develop/source/theme/)

that seems where it is searching:
```
make[1]: Entering directory '/home/pkguser/src/carla-git/src/carla/source/theme'
Generating moc_CarlaStyle.qt5.cpp
Generating moc_CarlaStyleAnimations.qt5.cpp
Generating moc_CarlaStylePlugin.qt5.cpp
CarlaStylePlugin.hpp:33: Error: Plugin Metadata file "carlastyle.json" does not exist. Declaration will be ignored
make[1]: Leaving directory '/home/pkguser/src/carla-git/src/carla/source/theme'
```

But after so much years without touch C or C++ and never QT, I can't understand CarlaStylePlugin.hpp (can be a environment variable trouble?):
```c++
#if (QT_VERSION >= QT_VERSION_CHECK(5, 0, 0))
    Q_PLUGIN_METADATA(IID "org.qt-project.Qt.QStyleFactoryInterface" FILE "carlastyle.json")
#endif
```

**Comment by mxmilkb** [comment]
N.b. https://travis-ci.org/falkTX/Carla works

**Comment by mxmilkb** [comment]
I created a `carla-aurcifix-git` PKGBUILD to copy carlastyle.json one directory up, but now that fails just after the previous error https://travis-ci.com/mxmilkb/aur-av-bin/builds/138323532

That log seems, if I'm reading it correctly, to expect bitmaps/style/arrow.png as source/theme/bitmaps/style/arrow.png rather than resources/bitmaps/style/arrow.png. Possibly it's the Makefile, but I'm not sure how that could go wrong?

**Comment by redtide** [comment]
It seems to me some issue with the RCC command within Docker that can't find files, see [my build here]. I think it's related with [this issue].

[my build here]: https://travis-ci.com/redtide/aurci/jobs/260855897
[this issue]: https://github.com/Framstag/libosmscout/pull/629#issuecomment-408052593

**Comment by localnet** [comment]
[Seems solved](https://travis-ci.org/localnet/aurci/builds/617660465), can you confirm @mxmilkb @redtide?

Thanks you @redtide by telling me the right direction.

**Comment by redtide** [comment]
YW.
Yes I got also my build ended successfully.

I see some additional branches there with a 1.1... before tagging a new release I would like to send you some minor changes I made.

Other than that I would like to know if you could agree by changing the README to point to some GitHub pages for instructions: [GitHub doesn't support variable substitution in README], but you can do it in `gh-pages` so you don't have to "hardcode" your own travis-ci project budge link so that people forking it will use yours and not theirs.
This way could be a workaround it and make automatically the substitution in the forked ones, so they shouldn't even change it.
I can do this for you as another PR, just let me know. In this case if localnet is a user account and not an organization you can just add a gh-pages branch and I'll do the rest.

[GitHub doesn't support variable substitution in README]: https://stackoverflow.com/questions/18673694/referencing-current-branch-in-github-readme-md

**Comment by localnet** [comment]
I don't have so much experience with Git, at this time I am doing commits to v1.1 branch to test it at Travis CI and after merge it in master branch (Maybe will be better if I rename v1.1 to develop)

**Comment by redtide** [comment]
A tag is just like a label over a commit, so you can do it also from GH by creating a new release from master after you merged all the needed changes to it.

**Comment by mxmilkb** [comment]
Resolved, thank you so much :)

**Comment by localnet** [comment]
Fixed 39fd6037a1ecb1c3190c8710c6b12fda85524c27
